clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/config/DockerRegistryConfigurationProperties.groovy

```diff
@@ -29,8 +29,10 @@ class DockerRegistryConfigurationProperties {
     String username
     // Docker registry password.
     String password
-    // File containing docker password.
+    // File containing docker password in plaintext.
     String passwordFile
+    // File containing docker's auth config (managed by docker-cli). Typically `~/.docker/config.json`.
+    String dockerconfigFile
```

Comment:
I wanted to be consistent with `kubeconfig` which is how it's capitalized in all the docs. I'll make sure `dockerconfig` is consistent here.